test/ICSharpCode.SharpZipLib.Tests/Zip/WindowsNameTransformHandling.cs

```diff
@@ -2,12 +2,20 @@
 using NUnit.Framework;
 using System;
 using System.IO;
+using System.Runtime.InteropServices;
 
 namespace ICSharpCode.SharpZipLib.Tests.Zip
 {
 	[TestFixture]
 	public class WindowsNameTransformHandling : TransformBase
 	{
+		[OneTimeSetUp]
+		public void TestInit() {
+			if (Path.DirectorySeparatorChar != '\\') {
```

Comment:
Is it worth using the [Platform] attribute so that the tests only run on Windows?

Reply:
Maybe, I didn't actually know about that one. I just figured feature detection was actually what was important anyway, instead of guessing based on the OS.

Reply:
I don't know what's best, just wondered if tests that involve drive letters might have issues as well as things with separators.

Reply:
Yeah maybe, it's kind of hard to test until any OS starts using drive letters and forward slashes though 😁